openwisp_users/tests/test_admin.py

```diff
@@ -493,6 +499,18 @@ def test_organization_uuid_field(self):
         html = '<input type="text" name="name" value="default"'
         self.assertNotContains(response, html)
 
+    def test_admin_actions(self):
+        request = MockRequest()
```

Comment:
Can you please explain this, Why is this an object of `MockRequest` which is empty! :smile: